examples/train_direction_reco.py

```diff
@@ -48,13 +49,22 @@
 )
 
 
+def save_results(db, tag, results, archive):
+    db_name = db.split("/")[-1].split(".")[0]
+    path = archive + "/" + db_name + "/" + tag
+    os.makedirs(path, exist_ok=True)
+    results.to_csv(path + "/results.csv")
+    print("Results saved at: \n %s" % path)
+    
```

Comment:
This seems duplicate of the function by the same name in [src/graphnet/models/training/utils.py](https://github.com/graphnet-team/graphnet/blob/97110e0d1286fdc9477cd49ef77f2b40963efcf0/src/graphnet/models/training/utils.py)

Reply:
I know. I started getting the pickle error that Moust mentioned. Using this removed the error.

Reply:
In that case I vote that we resolve #281 first, before adding more technical debt. 😅